cmake/ycm-0.2/find-modules/FindZFP.cmake

```diff
@@ -0,0 +1,37 @@
+#
+# Find the ZFP includes and library
+#
+
+# This module defines
+# ZFP_INCLUDE_DIR, where to find zfp.h
+# ZFP_LIBRARIES, the libraries to link against
+# ZFP_FOUND, if false, you cannot build anything that requires ZFP
+
+######################################################################## 
+
+if(WIN32)
+	set(CMAKE_DEBUG_POSTFIX "d") 
```

Comment:
Please see https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#cmake for details about the code style of the CMake files in YARP. 

In general, please avoid to use tabs (as their width depend on the program that is rendering the text), but always use spaces. Most existing IDEs and editors can be configured to emit spaces (2,4 or 8) when you press the tab key.